packages/vscode-extension/src/debug/teamsAppInstallation.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { VS_CODE_UI } from "../extension";
+import graphLoginInstance from "../commonlib/graphLogin";
+import * as commonUtils from "./commonUtils";
+import { ExtensionErrors, ExtensionSource } from "../error";
+import * as StringResources from "../resources/Strings.json";
+
+import axios from "axios";
+import { returnSystemError, returnUserError } from "@microsoft/teamsfx-api";
+
+export async function showInstallAppInTeamsMessage(detected: boolean): Promise<boolean> {
+  const message = `${
+    detected
+      ? StringResources.vsc.localDebug.installApp.detection
```

Comment:
Please extract the strings to package.nls.json and use localize() API to get string for later localization.

Reply:
Updated.